apps/design/frontend/vitest.config.ts

```diff
@@ -9,10 +9,15 @@ export default defineConfig({
 
     coverage: {
       thresholds: {
-        lines: 92,
-        branches: 72,
+        lines: 84.9,
```

Comment:
Guessing test files with higher coverage were skewing the higher before - dropped a bit after filtering out test files from coverage